arch/x86/include/asm/uaccess.h

```diff
@@ -11,6 +11,7 @@
 #include <asm/page.h>
 #include <asm/smap.h>
 #include <asm/extable.h>
+#include "linux/df-detection.h"
```

Comment:
Please move above, after linux/compiler.h and use angle brackets.

Overall, it is handy to sort the #includes alphabetically (see https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes for an example).
Linux kernel doesn't have strict policies regarding this, but we try to avoid increasing the entropy by keeping newly added headers sorted.